### PR TITLE
i18n(fr): Update a bunch of file from #8167 (STEPS)

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
 Update a bunch of file from #8167 (STEPS)

#### Related issues & labels (optional)
This is the continuation of [this PR about Steps](https://github.com/withastro/docs/pull/8208).
- Closes #
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/en/recipes/external-links.mdx
Comment (McFlyPartages) [inline src/content/docs/en/recipes/external-links.mdx:1]: Humm its not french translation ?

**Review by McFlyPartages — APPROVED** [review]
LGTM

File: src/content/docs/en/recipes/external-links.mdx
Comment (thomasbnt) [reply src/content/docs/en/recipes/external-links.mdx:1]: Yeah good point, removed this file.

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**